lib/collision_detector_2.cpp

```diff
@@ -176,6 +182,7 @@ ContinuousCollision CollisionDetector2::detectContinuousCollision(int entity_id,
     ContinuousCollision collision { opposite_opt, space_left_start_value };
 
     for (auto& [ opposite_id, opposite_collider ] : opposite_entities_with_collider) {
+        // TODO: If opposite_id = colliders creator_id (for bullets, when a player/weapon shoots, spawns a bullet, you should not collide with it)
```

Comment:
If this needs to get resolved, this is probably the best time to do so

Reply:
This is an outdated comment, we have already fixed this by using should_displace = false in the collider of the bullet and damage_system not dealing damage to yourself.